File: tests/rot.c
#include <stdio.h>

int main() {
    unsigned char buf[] =
    "\x0b\x1c\x2d\x3e\x4f\x60\x71\x82\x93\xa4\xb5\xc6\xd7\xe8"
    "\xf9\x0a";

    for (int i = 0; i < sizeof buf; i++) {
        buf[i] = (buf[i] - 11) & 0xff;
    }

    for (int i = 0; i < (sizeof buf - 1); i++) {
        printf("\\x%02x", buf[i]);
    }
    printf("\n");

    return 0;
}

File: tests/rot.cs
using System;

class Rot {
    static void Main() {
        byte[] buf = new byte[16] {0x0b,0x1c,0x2d,0x3e,0x4f,0x60,
        0x71,0x82,0x93,0xa4,0xb5,0xc6,0xd7,0xe8,0xf9,0x0a};

        for (int i = 0; i < buf.Length; i++) {
            buf[i] = (byte)((buf[i] - 11) & 0xff);
        }

        for (int i = 0; i < buf.Length; i++) {
            Console.Write(String.Format("0x{0:X2},", buf[i]));
        }
        Console.WriteLine();
    }
}